data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,57 @@ mission "Arachi Cartoonist 2"
+			`	Whimpering as the thunder grows closer once more, he spots a tree unlike the others, a great oak with branches thicker than the trees around him, so he goes to it. Laying down at the base of that largest tree, he falls asleep, and the curtain closes.`
+			`	It opens once again after a few moments, with the boy waking up in darkness. When he gets up, rumbling sounds are accompanied by rays of light reaching the ground, and he realizes the great oak had lowered its branches around him, and as it lifts them up once again, it knocks back several trees that had fallen on them during the storm. The sun as his guide, the boy manages to leave the forest, and the curtains close again.`
+				goto end
+			label pearl
+			`	Once you have the seashell in your hands, the organizers tell you to go through the path on the middle, and so you do.`
+			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it, while keeping your eyes on the stage in front of you, made of wood and about a third of the size of human opera stages. Several light rays shine onto it, and as you look up you can only barely see some spotlights high up on the trees. After the audience is all here, the curtains open, to reveal many small figures made out of what looks like paper, in a style that loosely resembles origami, and supported either by thin wooden poles from below, or strings on the top. There are figures close to where the curtains were, and others further and further deeper into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`
+			`	A Saryd is on top of some sort of long, wooden platform, with a deep blue background under it. Only when you see the fishing rod do you realize that he is on a boat, fishing at sea. He feels a bite, but once he reels it in there is nothing on the hook. He feels another bite, and again finds nothing. Time and again, he looks to the hook to find the bait still there, intact, but continues to feel the bites, that are sometimes so blatant even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`
+			`	Once it is night, the waves grow increasingly stronger, and the Saryd stops fishing altogether, putting the rod on the boat and trying to steer it back to shore, but he can't find the shore. A huge wave that goes beyond the stage's top approaches, and engulfs the boat.`
```

Comment:
Some sentences are quite long and unwieldy. Also, `steer it back to shore, but he can't find the shore.` sounds kind of repetitive. 
```suggestion
			`	Once it is night, the waves grow increasingly stronger, and the Saryd stops fishing altogether. He puts the rod on the boat and tries to steer it back to shore, but he can't find his way. A huge wave that goes beyond the stage's top approaches, and engulfs the boat.`
```